Tests/UITests/ResponseStubs/slide-up-close.json

```diff
@@ -25,8 +25,8 @@
                     "header": "Test Campaign",
                     "titleColor": "#000000",
                     "headerColor": "#000000",
-                    "messageBodyColor": "#000000",
-                    "backgroundColor": "#ffffff",
+                    "messageBodyColor": "",
```

Comment:
Why was this change needed?

Reply:
These changes add coverage to nil coalescing of [this](https://github.com/rakutentech/ios-inappmessaging/blob/ad881ea56ad734e8562bccab91a968461fbf83db/Sources/RInAppMessaging/Views/Presenters/SlideUpViewPresenter.swift#L25-L29) lines. ****